apps/api/v2/src/ee/event-types/event-types.repository.ts

```diff
@@ -55,7 +55,7 @@ export class EventTypesRepository {
         isUserOrganizationAdmin,
       });
     } catch (error) {
-      throw new NotFoundException(`User with id ${user.id} has no event type with id ${eventTypeId}`);
+      return null;
```

Comment:
don't we need to throw the error for our exception filters to catch ? or event just not put the try catch and let our exception filters handle it

Reply:
Fixed in [remove try catch](https://github.com/calcom/cal.com/pull/14446/commits/c02b2f2a8d164b6cc9ce82d95809924da31dce7e)